components/basic-hero/Component.jsx

```diff
@@ -13,11 +14,18 @@ import { Container } from "../../packages/grids/Container";
  */
 
 export default function BasicHero(props) {
-  const { title } = props;
+  const { title, titleAlignment } = props;
   return (
     <Container>
       <div className="su-flex su-justify-between su-flex-wrap su-rs-mt-6 su-rs-mb-5">
-        <h1 className="su-font-serif su-mb-0">{title}</h1>
+        <h1
+          className={cnb(
+            "su-font-serif su-mb-0",
+            titleAlignment === "center" && "su-text-center"
```

Comment:
```suggestion
            titleAlignment === "center" && "su-text-center su-mx-auto"
```
@iamrentman we might need an extra su-mx-auto class here for short page titles.

![Screenshot 2024-03-06 at 2 34 54 PM](https://github.com/SU-UComm/stanford-report/assets/42749717/47051ecf-bbf1-43d2-999d-ed89fb8b2097)

With su-mx-auto
![Screenshot 2024-03-06 at 2 35 07 PM](https://github.com/SU-UComm/stanford-report/assets/42749717/fa3e3b72-b719-4dbe-bdd9-c78df6633ab0)